src/resource/auth/auth.service.ts

```diff
@@ -81,6 +83,21 @@ export class AuthService {
     };
   }
 
+  async verifyUserEmail({ id }: VerifyEmailInput) {
+    const user = await this.userService.findById(id);
+    if (!user) {
+      this.logger.error(`User with id ${id} does not exist`);
+      return;
```

Comment:
`this.logger.error` will throw an exception when used, so we don't need to `return` here

Reply:
updated with throw error